scripts/getPDWorks.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+
+from model import Record
+from managers import DBManager
+
+import json
+
+jsonFile = 'UofM_FullAccess_CSV.json'
+
+fullAccessArray = []
+
+def main():
+
+    '''Retrieving former LA works that are now PD'''
+
+    dbManager = DBManager(
+        user= os.environ.get('POSTGRES_USER', None),
+        pswd= os.environ.get('POSTGRES_PSWD', None),
+        host= os.environ.get('POSTGRES_HOST', None),
+        port= os.environ.get('POSTGRES_PORT', None),
+        db= os.environ.get('POSTGRES_NAME', None)
+    )
+
+    dbManager.generateEngine()
+
+    dbManager.createSession()
+
+    with open('ingestJSONFiles/UofM_FullAccess_CSV.json') as f:
+        UofMBooks = json.load(f)
+
+    for record in dbManager.session.query(Record) \
+        .filter(Record.source == 'UofM').all():
+            for book in UofMBooks['data']:
+                if record.title in book['Title']:
+                    if book['Access in DRB'] == "Full Access":
+                        fullAccessArray.append(record.title)
+
+    print(fullAccessArray)
+    print(len(fullAccessArray))
```

Comment:
Are we just double checking here that we have the right records and number of records?

Reply:
These print statements are meant to double check how many records that were once in Limited Access from the Publisher's Backlist Project are now in Public Domain and what those records are.